tests/trace/test_objects_query_builder.py

```diff
@@ -174,7 +174,7 @@ def test_object_query_builder_sort():
         ) as version_count,
         row_number() OVER (
             PARTITION BY project_id, kind, object_id
-            ORDER BY (deleted_at IS NULL) DESC, created_at DESC
+            ORDER BY created_at DESC
```

Comment:
I would imagine this deleted clause was important. Can you share why it was removed here?

Reply:
For some reason this didn't work when I was developing. I recovered the clause now.